src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 4.4.0
 -----
 
+ * added the `impersonation_exit_url()` and `impersonation_exit_path()` functions that return url, that allow to switch back to the original user, when using user impersonation.
```

Comment:
added the `impersonation_exit_url()` and `impersonation_exit_path()` functions. They return a URL that allows to switch back to the original user.